### PR TITLE
Documentation Enhanced.

### PR DESCRIPTION
**Pull request by ljesparis**
Changes:

- Move the golang/gopath setup instructions into a full installation.md in the path
- Link from readme.md to the installation.md in section called installation/setup

The setup document must include
- instructions for setting up golang and gopath with gvm (linux)
- instructions for setting up golang and gopath with brew (OSX)
- instructions for installing golang on linux from binary and setting up gopath

### CODE REVIEW

File: Installation.md [hunk 1]
```diff
@@ -1,74 +1,119 @@
...
-
-Install `homebrew`, if you don't have it yet.
+First you need to have installed `homebrew`, if you don't have it yet.
 
 ```sh
 /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 ```
 
-Install the latest version of go
+Then, let's install latest go's version.
```
Comment (gz-c) [inline Installation.md]: go's latest version

File: Installation.md [hunk 2]
```diff
@@ -1,74 +1,119 @@
 # Installing go
 
 ## For OSX
-
-Install `homebrew`, if you don't have it yet.
+First you need to have installed `homebrew`, if you don't have it yet.
```
Comment (gz-c) [inline Installation.md]: have `homebrew` installed

File: Installation.md [hunk 3]
```diff
@@ -1,74 +1,119 @@
...
 ```sh
-bash < <(curl -s -S -L https://raw.githubusercontent.com/moovweb/gvm/master/binscripts/gvm-installer)
+sudo yum update -y && sudo yum upgrade -y
+sudo yum install -y git curl make gcc mercurial binutils bzr bison screen
+if [[ "$(cat /etc/redhat-release | grep -o CentOS)" == "CentOS" ]]; then sudo yum install -y build-essential libgmp3-dev; else sudo yum groupinstall -y "Development Tools" "Development Libraries" && sudo yum install -y gmp; fi;
+```	
+
+### Install Go with Gvm
+#### Install Gvm
+<strong>gvm</strong> need to be installed.
```
Comment (gz-c) [inline Installation.md]: `gvm`

File: Installation.md [hunk 4]
```diff
@@ -1,74 +1,119 @@
...
 ```sh
 gvm install go1.4 --source=https://github.com/golang/go
 gvm use go1.4
-gvm install go1.9
-gvm use go1.9 --default
 ```
 
-If you open up new terminal and the go command is not found then add this to `.bashrc`. GVM should add this automatically.
+#### Installation issues
+If you open up new terminal and the <strong>go command</strong> is not found then add this to `.bashrc`. GVM should add this automatically.
```
Comment (gz-c) [inline Installation.md]: use backquote ``` for <strong>

File: README.md [hunk 5]
```diff
@@ -46,21 +45,9 @@ Skycoin is small part of OP Redecentralize and OP Darknet Plan.
 
 ## Installation
 
-### Installing Go
+### Installation and Setup
```
Comment (gz-c) [inline README.md]: Keep comment about the required go version (1.9).  Go developers do not need to read installation instructions and only care which version is required.

Comment (ljesparis) [reply README.md]: Should specify `go` version at link or `Installation and Setup` subtitle?

Comment (gz-c) [reply README.md]: Both

**Pull request merged**